.github/workflows/commands.yml

```diff
@@ -4,7 +4,8 @@ on:
     types: [labeled, unlabeled]
   issue_comment:
     types: [created]
-
+concurrency:
```

Comment:
Not sure we have race conditions with with issues?

Reply:
Also not sure, but wouldn't hurt I guess